dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -198,24 +192,17 @@ public String getVersion(boolean withBuildNumber) {
     }
 
     /**
-     * The number of minutes for which a password reset token is valid. Can be
-     * overridden by {@link #PASSWORD_RESET_TIMEOUT_IN_MINUTES}.
+     * The number of minutes for which a password reset token is valid.
      */
-    public static int getMinutesUntilPasswordResetTokenExpires() {
+    public int getMinutesUntilPasswordResetTokenExpires() {
         final int reasonableDefault = 60;
-        String configuredValueAsString = System.getProperty(PASSWORD_RESET_TIMEOUT_IN_MINUTES);
-        if (configuredValueAsString != null) {
-            int configuredValueAsInteger = 0;
-            try {
-                configuredValueAsInteger = Integer.parseInt(configuredValueAsString);
-                if (configuredValueAsInteger > 0) {
-                    return configuredValueAsInteger;
-                } else {
-                    logger.info(PASSWORD_RESET_TIMEOUT_IN_MINUTES + " is configured as a negative number \"" + configuredValueAsInteger + "\". Using default value instead: " + reasonableDefault);
-                    return reasonableDefault;
-                }
-            } catch (NumberFormatException ex) {
-                logger.info("Unable to convert " + PASSWORD_RESET_TIMEOUT_IN_MINUTES + " from \"" + configuredValueAsString + "\" into an integer value: " + ex + ". Using default value " + reasonableDefault);
+        Integer configuredValue = settingsService.getValueForKeyAsInt(SettingsServiceBean.Key.MinutesUntilPasswordResetTokenExpires);
+        if (configuredValue != null) {
+            if (configuredValue > 0) {
```

Comment:
It is already by default 60 so it will always be true, so you can just return the MinutesUntilPasswordResetTokenExpires value. IMO I don't think we should be concerned by "malicious" activity, such as admin of the server removing the key or changing the value to negative.